packages/core/src/hooks/useUniqueId.ts

```diff
@@ -7,16 +7,13 @@ import * as React from "react"; // this can be optimized when react 17 support i
 const useReactId = (React as any)["useId".toString()] || (() => undefined);
 let count = 0;
 
-export const useUniqueId = (
-  deterministicId?: string,
-  idPrefix?: string
-): string => {
+export const useUniqueId = (deterministicId?: string) => {
```

Comment:
Since we are exporting this, I don't think we should remove this parameter.

Reply:
You're right - doesn't hurt to keep it. I've changed to keep `setUid` and deprecated it instead